docs/root/intro/arch_overview/upstream/load_balancing/load_balancers.rst

```diff
@@ -71,7 +71,16 @@ mapped onto a circle (the "ring") by hashing its address; each request is then r
 hashing some property of the request, and finding the nearest corresponding host clockwise around
 the ring. This technique is also commonly known as `"Ketama" <https://github.com/RJ/ketama>`_
 hashing, and like all hash-based load balancers, it is only effective when protocol routing is used
-that specifies a value to hash on.
+that specifies a value to hash on. If you want something other than the host's address to be used
+as the hash key (e.g. the semantic name of your host in a Kubernetes StatefulSet), then you can specify it 
+in the ``"envoy.lb"`` :ref:`LbEndpoint.Metadata <envoy_api_field_endpoint.LbEndpoint.metadata>` e.g.: 
+.. code-block:: yaml
```

Comment:
As long as you are here can you make this a validating-code-block? (See other examples). Same below.